src/Blocks/components/error/error.php

```diff
@@ -6,23 +6,26 @@
  * @package EightshiftForms
  */
 
-use EightshiftForms\Hooks\FiltersOuputMock;
+use EightshiftForms\Helpers\FormsHelper;
 use EightshiftFormsVendor\EightshiftFormsUtils\Helpers\UtilsHelper;
 use EightshiftFormsVendor\EightshiftLibs\Helpers\Helpers;
 
 $manifest = Helpers::getManifestByDir(__DIR__);
 
 $componentClass = $manifest['componentClass'] ?? '';
-$additionalErrorClass = $attributes['additionalErrorClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
 $selectorClass = $attributes['selectorClass'] ?? $componentClass;
 
 $errorValue = Helpers::checkAttr('errorValue', $attributes, $manifest);
 $errorId = Helpers::checkAttr('errorId', $attributes, $manifest);
+$errorTwSelectorsData = Helpers::checkAttr('errorTwSelectorsData', $attributes, $manifest);
+
+$twClasses = FormsHelper::getTwSelectors($errorTwSelectorsData, ['error']);
 
 $errorClass = Helpers::classnames([
-	FiltersOuputMock::getTwSelectors($componentClass, $attributes),
+	FormsHelper::getTwBase($twClasses, 'error', $componentClass),
 	Helpers::selector($selectorClass, $selectorClass, $componentClass),
-	Helpers::selector($additionalErrorClass, $additionalErrorClass),
+	Helpers::selector($additionalClass, $additionalClass),
```

Comment:
```suggestion
	$additionalClass,
```
Please do a find-replace for this in the whole plugin. The `classnames` handle will handle empty/null values.

Reply:
I will do this in the refactoring of the components with updates on the libs, I will skip this at this point.